cmd/utils/flags.go

```diff
@@ -293,8 +294,23 @@ var (
 	// Performance tuning settings
 	CacheFlag = cli.IntFlag{
 		Name:  "cache",
-		Usage: "Megabytes of memory allocated to internal caching (min 16MB / database forced)",
-		Value: 128,
+		Usage: "Megabytes of memory allocated to internal caching",
+		Value: 1024,
+	}
+	CacheDatabaseFlag = cli.IntFlag{
+		Name:  "cache.database",
+		Usage: "Percentage of cache memory allowance to use for database io",
+		Value: 75,
+	}
+	CacheGCFlag = cli.IntFlag{
+		Name:  "cache.gc",
+		Usage: "Percentage of cache memory allowance to use for trie pruning",
+		Value: 25,
+	}
+	CacheGCTimeoutFlag = cli.DurationFlag{
+		Name:  "cache.gc.timeout",
+		Usage: "Maximum time after which to flush memory-only tries to disk",
+		Value: 5 * time.Minute,
```

Comment:
I don't think we need CLI flags for such settings. You can always set them in the config file if you really want to tune details like this.

Reply:
Fixed, PTAL.